components/server/ee/src/gitlab/gitlab-app-support.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2020 Gitpod GmbH. All rights reserved.
+ * Licensed under the Gitpod Enterprise Source Code License,
+ * See License.enterprise.txt in the project root folder.
+ */
+
+import { ProviderRepository, User } from "@gitpod/gitpod-protocol";
+import { inject, injectable } from "inversify";
+import { TokenProvider } from "../../../src/user/token-provider";
+import { UserDB } from "@gitpod/gitpod-db/lib";
+import { Gitlab } from "@gitbeaker/node";
+
+@injectable()
+export class GitLabAppSupport {
```

Comment:
The separation in EE and non-EE  and the scope of the `GitLabService` binding makes it impossible to use that for the additions, unfortunately.